docs/flutter.md

```diff
@@ -246,3 +246,11 @@ dependencies {
 
 }
 ```
+
+## Demonstration of Live Preview with 360 Navigation
+
+<figure markdown>
+
+![livePreview with navigation](https://www.dropbox.com/s/plndb3g3w5ycu6k/streaming_screenshot.gif?raw=1){ loading=lazy }
```

Comment:
This displays locally for me when I run `mkdocs serve` I can see the navigation through the 360 live preview, and I can see Craig waving.

Reply:
oh, so it does work on the Mac.  nice.  Might be something specific to WSL.  It works on the live site.